src/loading.jl

```diff
@@ -26,13 +42,13 @@ If the desired device is CUDA (as determined by `ClimaComms.device()`), try load
 """
 macro import_required_backends()
     return quote
-        @static if $mpi_is_required() && $mpi_ext_is_not_loaded()
+        @static if $mpi_is_required()
```

Comment:
I've removed the `mpi_ext_is_loaded` check, since it is a no-op. Also, the function says that it will import the required.